client/src/Services/auth-services.js

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+
+axios.defaults.headers.common["Authorization"] = `Bearer ${localStorage.getItem(
+  "token"
+)}`;
+axios.defaults.headers.common["Content-Type"] = "application/json";
+
+async function getUser() {
+  const response = await axios.post("/api/user");
+
+  return response;
```

Comment:
fyi my IDE is saying this assignment to a variable is redundant. Looks like this works: 
```
async function getUser() {
  return await axios.post("/api/user");
}
```